app/views/people/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>The people of the Tasks!</h1>
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Incomplete Tasks</th>
+  </tr>
+  <tr>
+  <% @people.each do |person| %>
+
+    <td><%=link_to("#{person[:name]}", "/people/#{person.id}/show") %></td>
+
+   <% person_tasks = Task.where(:person_id => person.id) %>
```

Comment:
You should be able to combine these two variable definitions based on the fact that the tasks should be linked directly to a person.